### ISSUE TITLE
Issue Event Polling: 401 Unauthorized

### ISSUE THREAD

**Issue by JnCrMx** [JnCrMx/ChocoBot#18]
``javax.ws.rs.NotAuthorizedException: HTTP 401 Unauthorized`` when polling for issue events

probably thrown by 

https://github.com/JnCrMx/ChocoBot/blob/b2e6cf9f0ed03c0e0fb482ca7ece79727af03571/src/main/java/de/jcm/discord/chocobot/GitHubApp.java#L149

possible causes:
- wrong issue id
- wrong repo/user configuration (even though ?bugreport worked well)
- token generation failed or outdated token
- malformed request
- rate limit

***further investigation needed***

Stacktrace:
```
 javax.ws.rs.NotAuthorizedException: HTTP 401 Unauthorized
         at org.glassfish.jersey.client.JerseyInvocation.convertToException(JerseyInvocation.java:1056)
         at org.glassfish.jersey.client.JerseyInvocation.translate(JerseyInvocation.java:859)
         at org.glassfish.jersey.client.JerseyInvocation.lambda$invoke$1(JerseyInvocation.java:743)
         at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
         at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
         at org.glassfish.jersey.internal.Errors.process(Errors.java:205)
         at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:390)
         at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:741)
         at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:404)
         at org.glassfish.jersey.client.JerseyInvocation$Builder.get(JerseyInvocation.java:300)
         at de.jcm.discord.chocobot.GitHubApp.getIssueEvents(GitHubApp.java:159)
         at de.jcm.discord.chocobot.IssueEventRunnable.run(IssueEventRunnable.java:58)
         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
```

**Comment by JnCrMx** [comment]
Seems to be only a temporal inconvenience.
Closing for now.
Reopen if it occurs again.

**Comment by JnCrMx** [comment]
still occurring, look into GitHub's rate limits, ok?

**Comment by JnCrMx** [comment]
have a look at
https://developer.github.com/v3/#rate-limiting
and
https://developer.github.com/v3/#conditional-requests

**Comment by JnCrMx** [comment]
but with rate limits it would return 403 not 401...
maybe some problem about authorization tokens...
dunno

**Comment by JnCrMx** [comment]
I did a stupid mistake and mixed up `isAfter` and `isBefore`.
Fixed in 7ea2f6be09df34d98e26042c10927593d706d1db
